elliottlib/bzutil.py

```diff
@@ -290,6 +294,25 @@ def is_cve_tracker(bug_obj):
     return "SecurityTracking" in bug_obj.keywords and "Security" in bug_obj.keywords
 
 
+def get_valid_rpm_cves(bugs):
```

Comment:
I think you misunderstood the bug whiteboard here. IIUC ProdSec will label CVE trackers with their corresponding Brew component names, including both rpms and images. In Brew's convention, container image names are all ending with `-container`, while rpms are not. You can determine whether a CVE tracker should be attached to the rpm advisory or image advisory by checking its suffix.

Reply:
Thanks for this. I went with what's mentioned in the story, in case that's not correct or incomplete I can update it. @sosiouxme @smunilla

Reply:
Make sense!